packages/web-components/src/components/ic-popover-menu/ic-popover-menu.tsx

```diff
@@ -37,6 +37,44 @@ export class PopoverMenu {
   @Prop() anchor: string;
 
   /**
+   * The ID of the element that can open the popover menu
+   * if mouseAnchor is set to true. If null, then use the root element.
+   */
+  @Prop() clickEventEle: string = document.documentElement.id;
+  private clickEventEleRect: DOMRect = undefined;
```

Comment:
Please can you follow our [style guide](https://github.com/mi6/ic-ui-kit/blob/main/CONTRIBUTING.md#style-guide) and move the new private variables and props into the correct place in the file and in alphabetical order?